.gitlab-ci.yml

```diff
@@ -0,0 +1,71 @@
+stages:
+  - prepare
+  - test
+  - build
+
+variables:
+  GIT_DEPTH: 0
+
+default:
+  image: node:13
+  cache: &global_cache
+    key:
+      prefix: ${CI_PROJECT_PATH_SLUG}
+      files:
+        - yarn.lock
+    paths:
+      - node_modules
+    policy: pull
+initial_setup:
+  stage: prepare
+  cache:
+    <<: *global_cache
+    policy: pull-push
+  before_script:
+    - yarn install --ignore-engines --frozen-lockfile
+  script:
+    - node ./tools/scripts/calculate-commands.js $IS_PR > COMMANDS.txt
+    - echo $(cat COMMANDS.txt)
+  variables:
+    PREFIX: 'remotes/'
+    IS_PR: $CI_PIPELINE_SOURCE == "merge_request_event"
```

Comment:
`IS_PR` will become `merge_request_event == "merge_request_event"` and the actual call of the `calculate-commands` script is:

```bash
$ node ./tools/scripts/calculate-commands.js merge_request_event == "merge_request_event"
```

This caused me quite a lot of investigation in order to make the script work with an argument after `$IS_PR`.